app/lib/search/top_packages.dart

```diff
@@ -118,6 +118,7 @@ CachedValue<List<PackageView>> _cachedValue(
 }) {
   return CachedValue<List<PackageView>>(
     name: id,
+    // the redis-cached search results have 5 minutes TTL
```

Comment:
How does the 5 minutes relate to the 15 minutes interval?

Reply:
If we have a valid value locally, we don't initiate new search queries for the top packages for 15 minutes, and after that we have a good chance that we have a <5 minutes old value in the cache (we have 40-72 isolates asking for the  same data).

We could reduce the 15 minutes to 5 minutes, but in practice doesn't add more value, as the search index itself won't get updated only at every 15 minutes...

Reply:
Could you put these kind of arguments in the comments?

It currently reads a bit funny without the context:

```
// the redis-cached search results have 5 minutes TTL
interval: Duration(minutes: 15), // whyyy?
```